pyiso/base.py

```diff
@@ -426,7 +426,12 @@ def utcify_index(self, local_index, tz_name=None, tz_col=None):
                 aware_local_index = local_index.tz_localize(tz_name)
             except AmbiguousTimeError as e:
                 LOGGER.debug(e)
-                aware_local_index = local_index.tz_localize(tz_name, ambiguous='infer')
+                try:
+                    aware_local_index = local_index.tz_localize(tz_name, ambiguous='infer')
+                except AmbiguousTimeError as ate_two:
+                    LOGGER.warn('Second DatetimeIndex localization fallback, assuming DST transition day. ' + e.message)
+                    dst_active_list = self._dst_active_hours_for_transition_day(local_dt_index=local_index)
+                    aware_local_index = local_index.tz_localize(tz_name, ambiguous=dst_active_list)
```

Comment:
Lines 429-434 above are the fix for the ambiguous `01:00` time transition days out of daylight saving time, when there are 25 hours in the day.